dataland-e2etests/src/test/kotlin/org/dataland/e2etests/Constants.kt

```diff
@@ -1,3 +1,15 @@
 package org.dataland.e2etests
 
 const val BASE_PATH_TO_DATALAND_PROXY = "http://proxy:80/api"
+
+const val PATH_TO_KEYCLOAK_TOKENENDPOINT =
+    "http://proxy:80/keycloak/realms/datalandsecurity/protocol/openid-connect/token"
+
+const val TOKENREQUEST_GRANT_TYPE = "password"
```

Comment:
Auslagern in env Variablen?

Reply:
Die Werte hier dürften in allen Umgebungen konstant sein und beinhalten keine sensiblen Informationen.
Das Auslagern in envs würde meines Erachtens Dinge verkomplizieren.
Aber ich lasse mich gerne umstimmen wenn es gute Argumente dafür gibt :)

Reply:
Sehe ich auch so